lib/squeeze.ml

```diff
@@ -226,7 +230,7 @@ module Stuckness_monitor = struct
 				 let makingprogress = (delta_actual > 0L && direction = Some Down) || (delta_actual < 0L && direction = Some Up) in
 				 (* We keep track of the last time we were makingprogress. If we are makingprogress now 
 					then we are not stuck. *)
-				 if makingprogress then begin
+				 if makingprogress && not state.stuck then begin
```

Comment:
@robhoes If a VM doesn't balloon, we call it stuck in xapi. I'm not sure that if a VM, once stuck, can start ballooning properly without having to reboot the VM.